### DISCUSSION TITLE
Arabic Letter لا looks wrong on InDesign 19.0 - 19.4

### DISCUSSION THREAD

**Discussion by Mo7-For** [harfbuzz/harfbuzz#4779] [Q&A] [answered]
Hi
Since InDesign implemented this Engine the letter لا looked wrong it looks like لـا 

![image](https://github.com/harfbuzz/harfbuzz/assets/174314969/38a3a308-1e3b-47ee-b2e3-ee8b2d1680c3)

You can see the difference in the picture where is the up is wrong and using this engine 
and down is right and using the lagacy engine.
Please fix this.

**Comment by khaledhosny** [comment]
What font is this?

**Reply by Mo7-For** [reply]
(AH) Manal High

and it happens with these fonts as well:

(AH) Manal Bold
Al-Jazeera-Arabic
Al-Mohanad
Almudid
DecoType Naskh
Generator Black
KFGQPC_Uthman_Taha_Naskh
Khalid Art bold
mohammad bold art 1

**Reply by khaledhosny** [reply, marked as answer]
I tried the first three fonts but I couldn’t reproduce this issue in InDesign or elsewhere where HarfBuzz is used:
<img width="982" alt="image" src="https://github.com/harfbuzz/harfbuzz/assets/93914/505a6bc5-ede3-4945-b478-9ddd5145b143">

**Reply by Mo7-For** [reply]
The problem I'm dealing with is on my work computer, I've tried it now on my home PC and it works fine as you said.
What could be the cause of this problem? We're using the same Win11 and InDesign version.

**Reply by khaledhosny** [reply]
I have no idea. Check if yiu have ligatures disabled on the computer where this does not work, or if it is an old document, see ehat haopens if you create a new one. Either way, Adobe support forums is a more appropriate place to continue about this since there is no issue with HarfBuzz itself here.

**Reply by Mo7-For** [reply]
I double checked and I remembered that I used a script to turn off the engine on my PC, when I turned it back on it looked wrong again. 
Make sure you're not using the legacy engine, and your InDesign Version is 19.0+.
After enabling Harfbuzz copy and paste the text again and see if it still looks right, the text doesn't change unless you edit it.

https://github.com/harfbuzz/harfbuzz/assets/174314969/6f1ced36-3caa-4c51-a006-a095f5160e0e

**Reply by khaledhosny** [reply]
The font work fine in other applications that use HarfBuzz, you can even verify it yourself by dragging the font files to https://simoncozens.github.io/sploot/ and typing the same text.

The issue you are having might be an independent InDesign issue, or issue in how it uses HarfBuzz. Either case it is an InDesign bug, not a HarfBuzz one and there is no much we can do about it. You have to bring it with Adobe.

**Reply by khaledhosny** [reply]
BTW, when HarfBuzz is disabled, the ligatures option does not seem to affect lam-alef, but when HarfBuzz is enabled it does, so make sure you have it checked as this might be what is breakin the ligatures for you:

https://github.com/harfbuzz/harfbuzz/assets/93914/2c850c1a-a0ea-48ae-a885-d7ce61fbc779
